pkg/reconciler/pipelinerun/resources/resultrefresolution.go

```diff
@@ -173,6 +187,21 @@ func convertParams(params []v1beta1.Param, pipelineRunState PipelineRunState, na
 	return resolvedParams, nil
 }
 
+// convertWhenExpressions converts all When Expressions of the resolved pipeline run task
```

Comment:
we usually dont require docstrings for unexported functions but if we are going to have one here can you explain what "converts" means?